uqcsbot/utils/snailrace_utils.py

```diff
@@ -21,9 +21,9 @@
 # Messages
 SNAILRACE_ENTRY_ERR = "There is currently a race going on! Please wait until the current race finishes."
 SNAILRACE_ENTRY_MSG = "The race is currently open for entry! Entry is open for the next %s seconds" % SNAILRACE_OPEN_TIME
-SNAILRACE_ENTRY_CLOSE = "Entry is now closed. Let's Race!"
-SNAILRACE_JOIN = "%s has Joined the Race!"
```

Comment:
I thought this was a reference to smash bros and "X has Joined the Battle" (see [here](https://www.kapwing.com/explore/super-smash-bros-joins-the-battle-meme-template)), so I'm unsure about this change. Just my two cents, though.

Reply:
Fair, that one I'm pretty indifferent about